interface/modules/custom_modules/oe-module-ehi-exporter/templates/oe-module-ehi-exporter/ehi-exporter.html.twig

```diff
@@ -0,0 +1,116 @@
+                <ol>
+                    {% for tableResult in task.exportedResult.exportedTables %}
+                    <li>{{ tableResult.tableName|text}}: {{tableResult.count|text}}</li>
+                    {% endfor %}
+                </ol>
+                {% endif %}
+
+                {% if result.include_patient_documents %}
+                <p>{{ "Documents Exported"|xlt }}: {{ task.exportedResult.exportedDocumentCount|xlt }}</p>
+                {% endif %}
+
+            </details>
+            {% endfor %}
+
+            <a href="{{postAction|attr}}" class="btn btn-primary">{{ "Go Back"|xlt }}</a>
+        {% elseif result.getStatus() == 'failed' %}
+            <div class="alert alert-danger">
+                {{ "Check system logs as export failed to process"|xlt }}
+                <ul>
+                    {% for task in result.getJobTasks() %}
+                        {% if task.getStatus() == "failed" %}
+                        <li>{{ task.error_message }}</li>
+                        {% endif %}
+                    {% endfor %}
+                </ul>
+
+                {% if not memoryLimitUpdated %}
+                    {{ "The php.ini memory_limit setting could not be set to -1 to allow unlimited memory.  This could have caused your export to fail"|xlt }}
+                {% endif %}
+            </div>
+            <a href="{{postAction|attr}}" class="btn btn-primary">{{ "Go Back"|xlt }}</a>
+        {% endif %}
+    {% else %}
+        {% if errorMessage %}
+        <div class="alert alert-danger">
+            {{ errorMessage|text }}
+        </div>
+        {% endif %}
+        <h1 class="mt-3">{{ "Run Electronic Health Information Export"|xlt }}</h1>
+        <div>
+            <p>
+                {{ "Total Number of Patients in System"|xlt }}: <b>{{ exportSizeSettings.total_patients|text }}</b>
+            </p>
+        </div>
+        <div>
+            <p>{{ "The maximum file size stored is"|xlt }}: <b>{{ exportSizeSettings.max_document_size|text }}</b>.</p>
+            <p>{{ "Your server must have enough available memory (RAM) to hold twice this value if this value is greater than the Zip file limit setting. "|xlt}}
+        </div>
+        <!-- add form to run export.  Form needs to have an input checkbox to include documents, an input textbox to specify a patient pid -->
+        <div>
+            <p>
+                {{ "Current PHP Memory Limit"|xlt }}: <b>{{ exportSizeSettings.php_memory_limit|text }}</b>
+            </p>
+        </div>
+        <div>
+            <p>
+                {{ "Disk Free Space"|xlt }}: <b>{{ exportSizeSettings.disk_free_space|text }}</b>
+            </p>
+        </div>
+        {% if site_addr_oath is empty %}
+            test
+            <div class="alert alert-danger">
+                {{ "No Site Address Override has been configured for this installation.  Please set this address in order for the export readme to have the correct documentation links"|xlt }}
```

Comment:
only single spacing in translated constants

Reply:
Should be resolved now.